app/views/rating/applications/show.html.slim

```diff
@@ -43,6 +43,8 @@
         td = format_application_flags(@application)
 
   h2 Submitted Application
+  = show_student_names(@application)
```

Comment:
I personally don't see a huge benefit in outputting **both** names in the beginning and then just throwing in the whole application. In my opinion it would make more sense to have a layout like this:
```
h3 student0 name
panel
  heading question 1
  body answer 1
panel
  heading question 2
  body answer 2
h3 student1 name
panel
  heading question 1
  body answer 1
panel
  heading question 2
  body answer 2
```

But this might not be in the scope of this PR. Just came to my mind when I saw this line.